<launch>
    <node pkg="rr_platform" type="bigoli_steer_relay" name="steer_relay" output="screen" respawn="false">
        <param name="serial_port" value="/dev/arduino_steer"/>
        <param name="max_angle_msg_in" value="0.3"/>
        <param name="pid_p" value="0.7"/>
        <param name="pid_i" value="0.0"/>
        <param name="pid_d" value="2.0"/>
    </node>
</launch>
